app/javascript/mastodon/locales/zh-CN.json

```diff
@@ -77,9 +76,11 @@
   "navigation_bar.edit_profile": "修改个人资料",
   "navigation_bar.favourites": "赞的内容",
   "navigation_bar.follow_requests": "关注请求",
-  "navigation_bar.info": "关于本服务站",
+  "navigation_bar.info": "追加信息",
   "navigation_bar.logout": "注销",
-  "navigation_bar.mutes": "Muted users",
+  "navigation_bar.mutes": "被静音的用户",
+  // intentional departure from https://github.com/tootsuite/mastodon/blob/f864fee1/config/locales/zh-CN.yml#L126:
```

Comment:
Why the change to json? The zh-CN file used to have some react magic inherited from zh-HK that has to be run as actual executable code. And json does not play well with comments.